mage_ai/data_preparation/executors/block_executor.py

```diff
@@ -228,8 +228,6 @@ def execute(
                     delay=retry_config.delay,
                     max_delay=retry_config.max_delay,
                     exponential_backoff=retry_config.exponential_backoff,
-                    logger=self.logger,
```

Comment:
Did user mention the retry logging is confusing?

Reply:
no, but it just has a strange message in the log: `Exception thrown when attempting to run <function BlockExecutor.execute.<locals>.test.<locals>.__execute_with_retry at 0xffff8cde8280>, attempt 1 of 1`.

Reply:
I see... I think the main issue I'm trying to avoid is having multiple error logs for the same block since if I add the pipeline run error message, it will add another error log

Reply:
@wangxiaoyou1993 I changed the retry log to be a warning and added it back

Reply:
ok